### PR TITLE
Confdeck 139

### PR DESCRIPTION
**Pull request by beydogan**

### CODE REVIEW

File: app/assets/javascripts/conferences.js [hunk 1]
```diff
@@ -157,17 +157,17 @@ $(document).ready(function(){
...
+//            errorPlacement: function errorPlacement(error, element) {
+//                $(element).closest('.input-group').after(error);
+//            },
+//            highlight: function (element, errorClass, validClass) {
+//                $(element).closest("div.form-group").addClass("has-error").removeClass("has-success");
+//
+//            },
+//            unhighlight: function (element, errorClass, validClass) {
+//                $(element).closest("div.form-group").removeClass("has-error").addClass("has-success");
+//            },
```
Comment (xaph) [inline app/assets/javascripts/conferences.js]: @beydogan remove this comment outed lines

File: app/views/admin/layouts/_header.html.erb [hunk 2]
```diff
@@ -29,41 +29,42 @@
...
-              <a href="javascript:void(0);" data-toggle="collapse" data-target="#navbar-collapse">
-                    <span class="meta">
-                        <span class="icon"><i class="ico-sort-by-attributes-alt"></i></span>
-                    </span>
-              </a>
+                <a href="javascript:void(0);" data-toggle="collapse" data-target="#navbar-collapse">
+                        <span class="meta">
+                            <span class="icon"><i class="ico-sort-by-attributes-alt"></i></span>
+                        </span>
+                </a>
```
Comment (xaph) [inline app/views/admin/layouts/_header.html.erb:39]: @beydogan indentation problems?

File: app/views/admin/schedule/sidebar.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<div class="fixed">
+    <button type="button" class="btn btn-primary btn-block create-room-button mb15" ><%= t("rooms.create") %></button>
+
+    <div class="room-list">
+          <div class="panel panel-inverse">
```
Comment (xaph) [inline app/views/admin/schedule/sidebar.html.erb]: @beydogan indentation.

**Comment by xaph** [comment]
@beydogan  fix the indentations and comment outed blocks and merge it

**Comment by beydogan** [comment]
Conflict was solved. Merging.

**Pull request merged**